builtin/rebase.c

```diff
@@ -98,6 +98,7 @@ struct rebase_options {
 	int allow_rerere_autoupdate;
 	int keep_empty;
 	int autosquash;
+	int ignore_whitespace;
```

Comment:
Should I introduce an `enum {}` and `rebase_options->ws_flag` to keep track of all whitespace flags instead of having individual bools for each flag? Or is this a story for another patch? (Because we'll require one for `--whitespace=<...>` option?)

Reply:
I think this is a story for another patch.  If you look at the whitespace options in `git apply`, `--whitespace` and `--ignore-whitespace` are orthogonal, and can be used independently from each other.  And I assume it would be the same way in `rebase`.